src/settings-tab.ts

```diff
@@ -154,6 +168,7 @@ export class MindMapSettingsTab extends PluginSettingTab {
           })
       );
 
+    const color_2 =
     new Setting(containerEl)
```

Comment:
- move this new Setting to the previous line, right after the `=`, followed by a space char.